src/components/categories.js

```diff
@@ -0,0 +1,7 @@
+function Categories() {
```

Comment:
- [ ] Please, use ES6 best practices, in this case, this could improve being an arrow function. 🦊